resources/views/dashboard/admin/realops/index.blade.php

```diff
@@ -54,6 +54,7 @@
                     <td>
                         <a href="/dashboard/admin/realops/remove-pilot/{{ $f->id }}" class="btn btn-danger btn-sm float-right" title="Unassign Pilot" data-toggle="tooltip"><i class="fas fa-times"></i></a>
                         {{ $f->assigned_pilot->full_name }}
+                        <br/><small>({{ $f->assigned_pilot->email }})</small>
```

Comment:
Could we maybe put these into a div or a span so that they show up together? Right now, it's showing funny with the remove button. We could also reconsider how the remove button is done to make this look better.

![image](https://user-images.githubusercontent.com/38896380/217683284-7c2637b4-f11b-4d58-8d2d-4c8b7165f5e7.png)

Reply:
Fixed formatting issue using colspan technique in the table
![image](https://user-images.githubusercontent.com/21279582/218278277-e3d0aca9-d047-4e5f-8fe9-d509bd9f6a20.png)